google_assistant/run.sh

```diff
@@ -1,27 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bashio
 set -e
 
-CONFIG_PATH=/data/options.json
 CLIENT_JSON=/data/client.json
 CRED_JSON=/data/cred.json
 
-CLIENT_SECRETS=$(jq --raw-output '.client_secrets' $CONFIG_PATH)
-PROJECT_ID=$(jq --raw-output '.project_id' $CONFIG_PATH)
-MODEL_ID=$(jq --raw-output '.model_id' $CONFIG_PATH)
+PROJECT_ID=$(bashio::config 'project_id')
+MODEL_ID=$(bashio::config 'model_id')
 
 # check if a new assistant file exists
-if [ -f "/share/$CLIENT_SECRETS" ]; then
-    echo "[Info] Install/Update service client_secrets file"
-    cp -f "/share/$CLIENT_SECRETS" "$CLIENT_JSON"
+if bashio::config 'client_secrets'; then
```

Comment:
The clients' secrets are not in the config, but stored in a file. The config contains the file name. So in the location, it should check if the file (that is fetched from the config) exists.